### PR TITLE
Refactor integration test driver scripts.

### PR DESCRIPTION
**Pull request by coryan**
We need to have a cleaner way to separate the integration tests
for each library.  This is a first attempt.

### CODE REVIEW

File: bigtable/ci/run_integration_tests.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# This script should is called from the build directory, and it finds other
+# scripts in the source directory using its own path.
+readonly BINDIR=$(dirname $0)
```
Comment (mbrukman) [inline bigtable/ci/run_integration_tests.sh]: Quotes around RHS?

File: ci/build-docker.sh [hunk 2]
```diff
@@ -71,9 +71,11 @@ cd "${BUILD_DIR}"
 ctest --stop-on-failure
 
 # Run the integration tests.
-(cd bigtable/tests && /v/bigtable/tests/run_integration_tests_emulator.sh)
-(cd bigtable/tests && /v/bigtable/examples/run_examples_emulator.sh)
-(cd storage/tests && /v/storage/tests/run_integration_tests.sh)
+for subdir in bigtable storage; do
+  echo
+  echo "${COLOR_RESET}Running integration tests for ${subdir}"
```
Comment (mbrukman) [inline ci/build-docker.sh]: Why do we need an explicit color reset here? Shouldn't every script reset the color themselves? Or are you assuming something else might end the script with color set in the terminal to something? And is that considered a bug?

File: storage/ci/run_integration_tests.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# This script should is called from the build directory, and it finds other
+# scripts in the source directory using its own path.
+readonly BINDIR=$(dirname $0)
```
Comment (mbrukman) [inline storage/ci/run_integration_tests.sh]: Quotes around RHS?

File: ci/build-docker.sh (hunk 2 above)
Comment (coryan) [reply ci/build-docker.sh]: I was being paranoid.  Removed.

File: storage/ci/run_integration_tests.sh (hunk 3 above)
Comment (coryan) [reply storage/ci/run_integration_tests.sh]: Fixed.

**Comment by coryan** [comment]
@mbrukman PTAL.

**Pull request merged**